src/GameCoupling.ts

```diff
@@ -0,0 +1,103 @@
+export class Game {
+    private _lastSymbol: string = ' ';
+    private _board: Board = new Board();
+
+    public Play(symbol: string, x: number, y: number) : void {
+        //if first move
+        if (this._lastSymbol == ' ') {
+            //if player is X
+            if (symbol == 'O') {
+                throw new Error("Invalid first player");
+            }
+        }
+        //if not first move but player repeated
+        else if (symbol == this._lastSymbol) {
+            throw new Error("Invalid next player");
+        }
+        //if not first move but play on an already played tile
+        else if (this._board.TileAt(x, y).Symbol != ' ') {
+            throw new Error("Invalid position");
+        }
+
+        // update game state
+        this._lastSymbol = symbol;
+        this._board.AddTileAt(symbol, x, y);
+    }
+
+    public Winner() : string {
```

Comment:
**Message coupling:** we connect the class to the client using it exposing a method. The client would need to call (couple) this method. It is unavoidable and also the cleanest way to connect/couple two modules.